web/modules/custom/weather_block/weather_block.info.yml

```diff
@@ -3,3 +3,6 @@ type: module
 description: 'Displays weather for selected city'
 core_version_requirement: ^9 || ^10
 package: Custom
+
+libraries:
```

Comment:
I'm not even sure modules do anything with `libraries` configuration in info.yml